packages/webchat/src/main.tsx

```diff
@@ -148,6 +168,7 @@ class Web extends React.Component<MainProps> {
   loadOverrides(overrides: Overrides) {
     try {
       for (const override of _values(overrides)) {
+        // TODO: 15 - load module view this can't work
         override.map(({ module }) => this.props.bp!.loadModuleView(module, true))
```

Comment:
We will need to keep the load view logic. Custom components in the chat will use them, and we'll probably also keep the override mechanism for components, they will be bundled with the bot and served from the cdn